### PR TITLE
INT-393 - Add JAVA_OPTS to the application launch script

### PR DESCRIPTION
**Pull request by bigspotteddog**
https://issues.sonatype.org/browse/INT-393

### CODE REVIEW

File: templates/start-nexus-iq-server.sh.erb [hunk 1]
```diff
@@ -9,4 +9,4 @@
 %>
 
 cd <%= node['nexus_iq_server']['install_dir'] %>
-exec /usr/bin/java -jar <%= nexus_iq_jar  %> server <%= nexus_iq_config  %>
+exec /usr/bin/java -jar ${JAVA_OPTS} <%= nexus_iq_jar %> server <%= nexus_iq_config %>
```
Comment (fernau) [inline templates/start-nexus-iq-server.sh.erb]: it just looks odd to me that JAVA_OPTS is between -jar and the .jar file. according to the docs it should be `java [ options ] -jar file.jar`

**Comment by whyjustin** [comment]
Should have CI green on latest commit before merge.

**Comment by bigspotteddog** [comment]
CI is now green for this commit.

File: templates/start-nexus-iq-server.sh.erb (hunk 1 above)
Comment (bigspotteddog) [reply templates/start-nexus-iq-server.sh.erb]: Corrected the order.

**Pull request merged**